vmdb/app/models/ems_amazon.rb

```diff
@@ -35,7 +35,7 @@ def browser_url
   end
 
   def connect(options = {})
-    raise "no credentials defined" if self.authentication_invalid?(options[:auth_type])
+    raise "no credentials defined" if self.missing_credentials?(options[:auth_type])
```

Comment:
These new lines read much easier... and actually had the same exact meaning, but now the old confusing name is gone.